src/javascript/app/base/interview_popup.js

```diff
@@ -0,0 +1,37 @@
+const Cookies        = require('js-cookie');
+const BinarySocket   = require('./socket');
+const isEuCountry    = require('../common/country_base').isEuCountry;
+const Client         = require('../base/client');
```

Comment:
extra spaces before `=`, client should be on line 3 (base comes before common alphabetically)